admin_manual/maintenance/package_upgrade.rst

```diff
@@ -9,8 +9,11 @@ The best method for keeping ownCloud current on Linux servers is by configuring
 your system to use ownCloud's `Open Build Service`_ repository. Then stay 
 current by using your Linux package manager to install fresh ownCloud packages. 
 After installing upgraded packages you must run a few more steps to complete 
-the 
-upgrade. These are the basic steps to upgrading ownCloud:
+the upgrade. These are the basic steps to upgrading ownCloud:
+
+.. warning:: Make sure that you don't skip a major release when upgrading via repositories.
+   For example you can't upgrade from 8.1.x to 9.0.x directly as you would skip the 8.2.x
+   major release. Scroll down to Upgrading Across Skipped Releases for more info.
```

Comment:
One more item: in the second warning, 'Adressbook' should be 'Addressbook'.

You can create a :ref: directly to the 9.0 release notes like this. First create the :ref: in the release notes:

```
.. _9.0_release_notes_label:

Changes in 9.0
--------------
```
Then insert the reference in the Warning:

```
please have a look at the :ref:`9.0 release notes<9.0_release_notes_label>` for important information...
```

I created the custom label '9.0 release notes' in the above example because the default is to use the subheading, 'Changes in 9.1'. If you don't want a custom label for a :ref: then omit it, and the angle braces.